renderer/style.css

```
@@ -241,42 +241,64 @@ html, body {
+  margin-right: 6px;
+  /* 最長ラベル「入力待ち」基準で min-width を予約（idle 時もレイアウトを保つ） */
+  min-width: 6ch;
   transition: color 0.15s, background 0.15s;
+}
+
+.pane-status::before {
+  content: "";
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)